src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/ExplainIT.java

```diff
@@ -211,4 +211,37 @@ public void testContentTypeOfExplainRequestShouldBeJson() throws IOException {
 
         assertEquals("application/json; charset=UTF-8", response.getHeader("content-type"));
     }
+
+    @Test
+    public void assertExplainPrettyFormatted() throws IOException {
+        String query = String.format("SELECT firstname FROM %s", TestsConstants.TEST_INDEX_ACCOUNT);
```

Comment:
I think the general practice is to use our internal StringUtils.format() over String.format()

Reply:
Good point! Thanks!